pkgs/development/python-modules/numpy/default.nix

```diff
@@ -56,16 +56,14 @@ in buildPythonPackage (rec {
     # Patching of numpy.distutils is needed to prevent it from undoing the
     # patch to distutils.
     ./numpy-distutils-C++.patch
-  ]
-  ++ lib.optionals stdenv.cc.isClang [
+
```

Comment:
This may work but it is probably wrong. We definitely must do a cleanup PR after this.

Reply:
I don't understand you.  I thought this PR is the cleanup you wanted.

Reply:
If we’re removing the conditionals, shouldn’t the x86_64-darwin one also be removed?

Reply:
I considered that (as seen in the commit message), but I didn't expect we want to drop the test on other platforms.  And AFAIK that patch wasn't upstreamed, so the likelihood seems lower for the usual clash on update where reviewers don't drop a platform-specific patch.

Reply:
The patch uses `pytest.mark.skipif` conditional on platform, but if it’s also conditional in the derivation, that’s fine. That was just raised as a concern in the review of the other PR.

Reply:
Oh, I missed that.  So applied unconditionally now.

Reply:
I think I mistook this PR for the original and not the cleanup, upsi, sorry